src/pages/transactions/create/Create.tsx

```diff
@@ -85,33 +90,43 @@ export function Create() {
 
```

Comment:
`event` shouldn't be undefined/nullable here.

Reply:
removed